controls/roles/fastsync/tasks/main.yml

```diff
@@ -3,7 +3,8 @@
   get_url:
     url: "https://fastsync.stereum.cloud/{{ stereum.fastsync.network }}/{{ stereum.fastsync.service }}/{{ stereum.fastsync.service }}-latest.tar.gz"
     dest: "/tmp/{{ stereum.fastsync.service }}-latest.tar.gz"
-    mode: '0644'
+    mode: '0600'
```

Comment:
The [DOC](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/get_url_module.html) says you need to get rid of the `'` characters so it is `0600` or you can do `'600'`.